content/blog/tidymodels-survival-analysis/index.Rmd

```diff
@@ -0,0 +1,129 @@
+
+### Add survival analysis to your tool kit
+
+If you are already familiar with tidymodels but maybe not survival analysis, these changes let you leverage the familiar framework for an additional type of modeling problem. Survival analysis offers methods for modeling time-to-event data. While it has its roots in medical research, it has broad applications as that event of interest can be so much more than a medical outcome. Take customer churn as an example: We are interested in how long someone is a customer for and when they churn. For customers who churned, we have the complete time for which they were customers. For existing customers, we only know how long they've been customers for _so far_. Such observations are called censored. So what are our modeling choices here? 
+
+We could look at the time and model that as a regression problem. We could look at the event status and model that as a classification problem. Both options might get us somewhere close to an answer to our original modeling question but not quite there. Survival analysis with its censored regression models lets us model an outcome that includes both aspects, the time and the event status. And with that, it can deal with both censored and uncensored observations appropriately. With this type of model, we can predict the survival time, or in more applied terms, how long someone will stay a customer for overall. We can also predict the probability of survival at a given time point. This let's us answer questions like "How likely is it that this customer will churn after 3 months?". See which prediction types are available for which models at [censored.tidymodels.org](https://censored.tidymodels.org/).
+
+
+## Ch-ch-changes: What's new for censored regression?
+
+The main components needed for this full-fledged integration of survival analysis into tidymodels were
+
+- Survival analysis models that can take censoring into account
+- Survival analysis performance metrics that can take censoring into account
+- Integrating changes required by these models and metrics into the framework
+
+For the models, parsnip gained a new mode, `"censored regression"`, for existing models as well as new model types such as `proportional_hazards()`. Engines for these reside in the parsnip extension package censored. This has been around for a while and we've previously shared [our inital thoughts](https://www.tidyverse.org/blog/2021/11/survival-analysis-parsnip-adjacent/) and the [release of censored](https://www.tidyverse.org/blog/2022/08/censored-0-1-0/).
```

Comment:
I tripped up on "parsnip extension package censored," though I'm not sure how we could write this clearly while adhering to tidy style.

Reply:
had another go at it :)